rmgpy/data/kinetics/family.py

```diff
@@ -4670,3 +4673,55 @@ def _child_make_tree_nodes(family, child_conn, template_rxn_map, obj, T, nprocs,
                            extension_iter_max=extension_iter_max, extension_iter_item_cap=extension_iter_item_cap)
 
     child_conn.send(list(family.groups.entries.values()))
+
+def average_kinetics(kinetics_list):
+    """
+    Based on averaging log k.
+    Hence we average n, Ea, arithmetically, but we
+    average log A (geometric average)
+    """
+    logA = 0.0
+    n = 0.0
+    Ea = 0.0
+    count = 0
+    for kinetics in kinetics_list:
+        count += 1
+        logA += np.log10(kinetics.A.value_si)
+        n += kinetics.n.value_si
+        Ea += kinetics.Ea.value_si
+
+    logA /= count
+    n /= count
+    Ea /= count
+    Aunits = kinetics_list[0].A.units
+    if Aunits == 'cm^3/(mol*s)' or Aunits == 'cm^3/(molecule*s)' or Aunits == 'm^3/(molecule*s)':
```

Comment:
All of these statements should look like `if Aunits in ("cm^3/(mol*s), (cm^3/(molecule*s)), ...)` rather than the big or piles.

Reply:
Note use of tuple, not list.

Reply:
I was curious...
<img width="869" alt="Screenshot 2023-07-14 at 2 09 45 PM" src="https://github.com/ReactionMechanismGenerator/RMG-Py/assets/93807/e9f70767-ead7-4ef9-b102-f1754e27357b">

Reply:
This isn't a speed thing as much as an overly strict linting rule - see [pylint's R1714](https://pylint.pycqa.org/en/latest/user_guide/messages/refactor/consider-using-in.html) - that I just happen to really like. I was actually wrong though, we should be using a set which should run even faster.

Reply:
You're right that `in set` is faster still.